### DISCUSSION TITLE
What's the meaning of the two rows in the trail/grid view

### DISCUSSION THREAD

**Discussion by claremacrae** [SkepticMystic/breadcrumbs#233] [Q&A] [answered]
Hallo! I'm experimenting with the Trail/Grid view, having set up a few test files in a test vault with `BC-folder-note: down` in the front matter, and it's looking super promising for navigation around my real vault...

I always seem to get these two rows in the Trail view, and I can't figure out how they differ - they always seem to have the same content.

![image](https://user-images.githubusercontent.com/4840096/147888468-4755f921-50ab-4713-ae27-da6ab26f235e.png)

Do they differ in any way?
Is it possible to see only one of them, if they are the same.
Should I prefer one over the other?

Thanks in advance!

**Comment by BricoleurSoul** [comment]
The two are simply different presentations of the same information.  You can choose which one (or both) to show through switches in the plug in options. If you are using the <previous | next> functionality you can also show those entries at the top of the note with the third switch. I personally prefer to show the grid and previous|next information (but it really is up to each users personal preference).

The switch is found in the Breadcrumb option as shown in the screenshot of my setup: 
<img width="956" alt="image" src="https://user-images.githubusercontent.com/43626234/147899670-e195ea68-0a12-4325-89cc-63b2fbf65bb6.png">

**Reply by SkepticMystic** [reply]
Exactly, yes!
They show exactly the same information. One small difference is that, with the top version (arrow-separated), you can click a button to show All paths, or only the Shortest path. This is not available with the grid view

**Reply by BricoleurSoul** [reply]
sorry! I'd forgotten this sometimes (ymmv and you'll find it an "always") useful option!

**Reply by claremacrae** [reply]
> Exactly, yes!
> They show exactly the same information. One small difference is that, with the top version (arrow-separated), you can click a button to show All paths, or only the Shortest path. This is not available with the grid view

I'm not seeing that difference - and not seeing that button. How can `BC-folder-note` give different routes?

**Comment by claremacrae** [comment]
Thanks everyone for the feedback...

> The two are simply different presentations of the same information. You can choose which one (or both) to show through switches in the plug in options

I'm struggling to understand how the settings option gives me that choice.

With this turned on:

![image](https://user-images.githubusercontent.com/4840096/147936075-0c5e718a-6d9a-442d-a526-0eb33c6be980.png)

I see both:

![image](https://user-images.githubusercontent.com/4840096/147936402-4aaab434-ba92-46ee-8146-512496e344f8.png)

With this turned off:

![image](https://user-images.githubusercontent.com/4840096/147936301-19735198-c249-412c-bb27-829ecf331a8d.png)

I see neither:

![image](https://user-images.githubusercontent.com/4840096/147936278-54e50ba6-8fab-45b5-95ac-9e6a2c44c726.png)

**Reply by claremacrae** [reply]
> Thanks everyone for the feedback...
> 
> > The two are simply different presentations of the same information. You can choose which one (or both) to show through switches in the plug in options
> 
> I'm struggling to understand how the settings option gives me that choice.

Cancel that - I now see it's this:

![image](https://user-images.githubusercontent.com/4840096/147936763-48a942da-4b2b-4d86-811d-831b352a5ca3.png)

You have to mouse-over the checkboxes to see what they do...

This is one of those things that is obvious once you know what the answer is...

**Comment by claremacrae** [comment, marked as answer]
Here is an update of @BricoleurSoul 's [image above](https://github.com/SkepticMystic/breadcrumbs/discussions/233#discussioncomment-1898130), that shows which of the options controls which elements are displayed at the top of the note:

![image](https://user-images.githubusercontent.com/4840096/147937195-a164caad-4737-4b04-959e-77d8d252369b.png)

**Reply by SkepticMystic** [reply]
Yes, it is difficult to tell... I'm not sure how to give them a label without making 3 separate settings, though

**Reply by claremacrae** [reply]
> Yes, it is difficult to tell... I'm not sure how to give them a label without making 3 separate settings, though

I'm not sure what the solution is - if I could have come up with better wording, I would have suggested it...

I don't think it's worth changing, but one idea might be a project vocabulary/glossary page, for the different parts of the plugin...
As well as helping users, it might help reveal areas where there might be confusion...

For example, is the Trail View a view? It certainly seems to be.
And yet, the option I marked in read says "Views to show" - and I think that may be what confused me...
So are Trail, Grid and Next-Previous also views?
Or are they sub-views?
Or are they components of the Trail View?
And how does Grid fit in to this?

This is all rhetorical - just meant to encourage thoughts, not answers!

I'm finally starting to "grok" how I might use this on a large vault...

Just one more question to formulate, separately, first...

**Reply by claremacrae** [reply]
I was trying to say "thank you very much for an amazing plugin!"

**Reply by SkepticMystic** [reply]
I think that's a great idea! I've actually already got a (basically empty) [terminology](https://github.com/SkepticMystic/breadcrumbs/wiki/Terminology) section in the wiki, I'll add some more content to it :)

**Comment by michaelpporter** [comment]
Closing — the trail/grid view was reworked in v4, so this no longer applies. Reopen if the question resurfaces.
